packages/aragon-client/src/index.js

```diff
@@ -36,6 +36,45 @@ class AppProxy {
     ).pluck('result')
   }
 
+  /**
+   * Create a handle to an external contract.
+   *
+   * @param  {string} address The address of the external contract
+   * @param  {Array<Object>} jsonInterface The JSON interface of the external contract
+   * @return {Object}
+   */
+  external (address, jsonInterface) {
+    let contract = {
```

Comment:
Could be `const`?

Reply:
yeah that's something I missed while iterating a bit